### PR TITLE
feat: Search + Explain with Navie

### PR DESCRIPTION
**Pull request by kgilpin**

### CODE REVIEW

File: src/webviews/getWebviewContent.ts [hunk 1]
```diff
@@ -28,12 +29,6 @@ export default function getWebviewContent(
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="Content-Security-Policy" content="
```
Comment (kgilpin) [inline src/webviews/getWebviewContent.ts:31]: @dividedmind as our resident security person, and I think you added this initially, why was this added? We need to allow the UI to invoke JSON-RPC to our services.

Comment (dividedmind) [reply src/webviews/getWebviewContent.ts:31]: This is to prevent the webview content from being coerced to load external resources, which could cause data leak or worse. I don't think it's a good idea to remove this altogether; instead a directive like `connect-src http://localhost:12345;` should be added. Although ideally I think the UI shouldn't talk to the services directly at all; instead the extension should intermediate.

**Comment by kgilpin** [comment]
See https://github.com/getappmap/vscode-appland/pull/859
